src/Lorentz/Contracts/BaseDAO/TZIP16Metadata.hs

```diff
@@ -93,13 +92,6 @@ knownBaseDAOMetadata settings = mconcat
 -- Off-chain views
 ------------------------------------------------------------------------
 
--- | A version of 'compileViewCode' that is simpler to use.
---
--- We cannot use 'compileViewCodeTH' easily because this way passing config
--- won't work.
-compileViewCode_ :: ViewCode st ret -> CompiledViewCode st ret
```

Comment:
There's a copy of this function (and an usage of it) in the `Lorentz.Contracts.RegistryDAO`) that should be fixed as well.